modules/node/handler/node_form_response.e

```diff
@@ -228,6 +228,12 @@ feature -- Form
 			ts.set_default_value ("Preview")
 			f.extend (ts)
 
+			if a_node /= Void and then a_node.id > 0 and then has_permission ("delete " + a_name) then
```

Comment:
we may need to distinguish

"delete own " + a_name
and 
"delete any " + a_name